docs/components/chips/README.md

```diff
@@ -327,7 +327,7 @@ Filter chips that directly filter content use the default chip style.
 
```

Comment:
select seems like the big category for things that may be triggered without a submit
option group
toggle
switch
tab and accordion (control)


to me this example is most close to an option group or list group and I wonder if there is a better name than selection chips?

Reply:
How about.... "choice" chips?